### DISCUSSION TITLE
Which programming languege is popular in NAIST?

### DISCUSSION THREAD

**Discussion by c3pk1** [NAIST-SE/AttractingContributors#61] [General]
https://github.com/c3pk1/AttractingContributors-Programminglanguages
I want to know which programming languege is popular in NAIST, so please tell me it freely using Pull Request :)

**Comment by 278Mt** [comment]
added! and plz help my favorite movies!
https://github.com/278mt/AttractingContributors
thx;-)

**Comment by sergiodl-naist** [comment]
Added one more! I don't think is popular but I wonder if there's people using it.
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by iikyara** [comment]
I added Flutter!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Comment by zhangyongxin1212** [comment]
https://github.com/zhangyongxin1212/AttractingContributors-Game/pulls
added and plz help me
